pkg/daemon/ceph/osd/volume.go

```diff
@@ -35,13 +35,14 @@ import (
 )
 
 const (
-	osdsPerDeviceFlag    = "--osds-per-device"
-	crushDeviceClassFlag = "--crush-device-class"
-	encryptedFlag        = "--dmcrypt"
-	databaseSizeFlag     = "--block-db-size"
-	dbDeviceFlag         = "--db-devices"
-	cephVolumeCmd        = "ceph-volume"
-	cephVolumeMinDBSize  = 1024 // 1GB
+	osdsPerDeviceFlag      = "--osds-per-device"
+	crushDeviceClassFlag   = "--crush-device-class"
+	crushDeviceClassHybrid = "hybrid"
```

Comment:
What does hybrid mean? We are just forcing it to this value so we can use the desired metadata device even if it appears as an hdd in the properties?

Reply:
Currently, the CRUSH device class property of an OSD is detected by the OSD itself, this value is based on the rotational property of the drive, so it's either `hdd` or `ssd`. Based on that information, Ceph will do some auto-tuning. Setting a new class `hybrid` tells Ceph it's a different OSD setup (potential faster than the previous 2).

@mmgaggle anything to add to that?